test/unit/models/consent.test.ts

```diff
@@ -0,0 +1,207 @@
+/*****
```

Comment:
there should be `model` not `models` folder name -> folder name should be the same as in src

Reply:
Resolved in a recent commit.